problem_builder/public/js/mentoring_standard_view.js

```diff
@@ -35,10 +35,13 @@ function MentoringStandardView(runtime, element, mentoring) {
             }
         }
 
+        // Data may have change, we have to re-validate.
+        validateXBlock();
+
```

Comment:
@mtyaka `change` ⟶ `changed`. Also, is this still necessary now that answer blocks are handling validation themselves?